oceannavigator/frontend/package.json

```diff
@@ -62,6 +57,8 @@
     "mini-css-extract-plugin": "^1.6.2",
     "node-sass": "^4.13.1",
     "sass-loader": "^6.0.6",
+    "ts-loader": "^8.2.0",
```

Comment:
Can't update to latest v9 cuz it doesn't support webpack 4 which is what we're currently on. no big deal tho https://github.com/TypeStrong/ts-loader/issues/595#issuecomment-827253722